shell/platform/tizen/channels/app_control.cc

```diff
@@ -281,7 +316,7 @@ AppControlResult AppControl::Reply(std::shared_ptr<AppControl> reply,
     return AppControlResult(APP_CONTROL_ERROR_INVALID_PARAMETER);
   }
   AppControlResult ret = app_control_reply_to_launch_request(
-      reply->Handle(), this->handle_, result_e);
+      reply->handle(), this->handle_, result_e);
```

Comment:
Is there no need the null checking for `AppControl*`? other functions have same issues.

Reply:
https://github.com/swift-kim/plugins/blob/ffe8d2536693eb1d222cb354b5b958813662f654/packages/tizen_app_control/lib/app_control.dart#L64-L93

An exception will be thrown in the Dart side constructors in case of instantiation failure so it cannot be nullptr as far as I can see (although it can be "invalid" if the destructor already has been called. The platform will return error if the app_control handle is invalid.)

Also I'll closely review the overall implementation in a follow-up PR. I tried not to make many changes to the existing code in this PR.

Reply:
ok I understand.